src/startscene.js

```diff
@@ -0,0 +1,192 @@
+import {
```

Comment:
Might make sense at a later stage to separate those functions in different files with different responsibilities. But for this state it is perfectly fine.

Reply:
I've evolved this slightly in some early commits in the ch4 branch, was thinking maybe of bringing them over here into this one. Might put something in here to see what it would look like